components/automate-ui/src/app/components/pending-edits-bar/pending-edits-bar.component.ts

```diff
@@ -52,9 +62,10 @@ export class PendingEditsBarComponent implements OnDestroy {
 
   public updateDisplay() {
     this.layoutFacade.layout.userNotifications.pendingEdits =
-      this.projectsHaveStagedChanges
-      || this.updateProjectsCancelled
-      || this.updateProjectsFailed;
+      this.isAuthorized &&
```

Comment:
... and use that result here. See note under `isBarHidden` as to why it needs to be precisely here.